packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -62,7 +63,7 @@ export class AccordionItem extends LitElement {
   /** Specifies a description for the component. */
   @property() description: string;
 
-  /** When `true`, the component is expanded. */
+  /** When `true`, the component is expanded to show child components. */
```

Comment:
Do we need the `to show child components`? In some cases its just text content or HTML elements and not components. Applies to other components as well. In the event we refer to it as "content" but here we say "child components" cc @geospatialem

Reply:
Concur with Matt's thought above, could we keep the messaging a bit more broad to content across the components?:

```suggestion
  /** When `true`, expands the component and its contents. */
```